galite-tests/src/test/kotlin/org/kopi/galite/tests/dsl/FormDSLTests.kt

```diff
@@ -303,6 +317,12 @@ class FormWithMultipleBlock : Form() {
     icon = "break"
   }
 
+  val resetFormCmd = command(item = resetForm) {
```

Comment:
Can we move the current form (along with the other forms in this file) into a separate file

Reply:
this form is related to this class test